hack/run-functests.sh

```diff
@@ -29,7 +29,7 @@ fi
 echo "Running local tests"
 FOCUS=$(echo "$FEATURES" | tr ' ' '|') 
 echo "Focusing on $FOCUS"
-if ! GOFLAGS=-mod=vendor ginkgo --focus=$FOCUS functests -- -junit /tmp/artifacts/unit_report_local.xml; then
+if ! GOFLAGS=-mod=vendor ginkgo --focus=$FOCUS functests -- -junit /tmp/artifacts/unit_report_local.xml -report ./tmp/artifacts/report.log; then
```

Comment:
may add `local` to the report filename, similar to the unit filename? We might get more of them if we use a custom reporter in the external repos as well.

Reply:
Will change, but my idea is to incorporate externals as vendored dependencies (see https://github.com/openshift-kni/cnf-features-deploy/pull/138 ) so on the mid-term I expect to have only one test suite.